README.md

```diff
@@ -44,6 +44,61 @@ Include the download centre nav-menu:
     ...
 ```
 
+## WS mode
+Package can be used with websockets to provide asynchronous communication
+between frontend and backend.
+If package will be used in websockets mode additional settings must be applied.
+* Inside common settings `WS_MODE` must be toggled on, and master url need to be provided.
+    ```
+    ASYNC_DOWNLOADS_WS_MODE = True
+    ASYNC_DOWNLOADS_SITE_URL = 'http://example.com'
+    ```
+* Application need to be configured as a `ASGI` and proper WS urls need to be configured:
+    ```
+    # ws/urls.py
+    from async_downloads.ws_consumers import DownloadsConsumer
+     
+     urlpatterns += [
+        re_path(r"ws/downloads/(?P<username>[\w.]+)/$", DownloadsConsumer.as_asgi()),
+     ]
+    ```
+    ```
+    # example asgi.py
+    from django.core.asgi import get_asgi_application
+
+    from channels.auth import AuthMiddlewareStack
+    from channels.routing import ProtocolTypeRouter, URLRouter
+
+    import ws.urls
+
+    application = ProtocolTypeRouter(
+        {
+            "http": get_asgi_application(),
+            "websocket": AuthMiddlewareStack(URLRouter(ws.urls.urlpatterns)),
+        }
+    )
+    ```
+
+* Change in application template async-downloads section to use WS js version.
+    ```
+    {{ request.user.get_username|json_script:"username" }}
+    {{ SITE_URL|json_script:"SITE_URL" }}
+    <script src="{% static "js/ws_async_downloads.js" %}" id="async-downloads-script"
+        data-url="{% url 'async_downloads:ajax_update' %}"
+    </script>
```

Comment:
```suggestion
    <script src="{% static "js/ws_async_downloads.js" %}" id="async-downloads-script"
        data-url="{{ FULL_WS_URL }}"
    </script>
```

Reply:
There still remain need for 'SITE_URL' if we connect to WS in different app 5e41739